.changeset/green-files-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Remove outdate Badge component export
```

Comment:
Thanks for catching this @liruifengv! I think we can tweak this a little bit for clarity:

```suggestion
Removes an outdated export in `package.json`
```